CHANGELOG.md

```diff
@@ -192,6 +192,9 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
   let didUpdateObject: (any P) -> ((Int) -> Void)? = P.didUpdateObject
   ```
+  
+  > **Note**  
```

Comment:
Can you add an example using #selector since that was the case we saw in the wild?